osgbuild/svn.py

```diff
@@ -152,31 +152,52 @@ def is_restricted_target(target):
     return False
 
 def restricted_branch_matches_target(branch, target):
-    """Return True if the pattern that matches 'branch' is associated with the
-    same name (e.g. 'main', 'upcoming', 'versioned') as the pattern that
-    matches 'target'; False otherwise.
+    """Return True if the pattern that matches `branch` is associated with the
+    same name (e.g. 'devops', 'main', 'upcoming', 'versioned') as the pattern that
+    matches `target`; False otherwise.
```

Comment:
what are these backticks doing here?!?!?!?

Reply:
that seems to be the convention for marking something as a symbol instead of a string literal.

Reply:
Ah ok, sorry for the noise!

I didn't understand why it was changing and i noticed some things were with single-quotes and some with backticks.

Anyway makes sense now!